app/Http/Controllers/SongController.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Song;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+use Tymon\JWTAuth\Facades\JWTAuth;
+
+class SongController extends Controller
+{
+    /**
+     * @var
+     */
+    protected $user;
+
+    /**
+     * SongController constructor.
+     */
+    public function __construct()
+    {
+        $this->user = JWTAuth::parseToken()->authenticate();
+    }
+
+    /**
+     * Display a listing of songs.
+     *
+     * @return Response
+     */
+    public function index()
+    {
+        $songs = $this->user->songs()->get(['artist', 'title', 'cues'])->toArray();
```

Comment:
Aren't we supposed to get **all** songs here, not limited to only logged in user songs?